### ISSUE TITLE
the alpha version was out of date

### ISSUE THREAD

**Issue by alexwhen** [YungSang/coreos-packer#5]
the Official alpha version has been updated,can i make it by offical source which is https://coreos.com/docs/running-coreos/platforms/vagrant ? how can i do?

**Comment by andyshinn** [comment]
Yep, there is also a nasty bug for some systemctl stuff in Vagrant due to https://coreos.com/releases/#549.0.0. @YungSang can we get an updated alpha, please?

**Comment by ailispaw** [comment]
https://github.com/AntonioMeireles/coreos-packer
https://atlas.hashicorp.com/AntonioMeireles/

**Comment by ryzy** [comment]
Does it mean @YungSang you're not doing any more up-to-date releases and one should switch to @AntonioMeireles images?

**Comment by alexwhen** [comment]
@ryzy maybe you are right

**Comment by andyshinn** [comment]
For anyone else who might be following this hoping for an update, it looks like YungSang is done maintaining these: https://github.com/YungSang/boot2docker-vagrant-box/issues/22#issuecomment-74710458.

I don't have an alternative yet, unfortunately :(

**Comment by AntonioMeireles** [comment]
fwiw i've been maintaining, since @YungSang departure both **coreos-packer** and **boot2docker-vagrant-box**  at **[AntonioMeireles/coreos-packer](https://github.com/AntonioMeireles/coreos-packer)** and **[AntonioMeireles/boot2docker-vagrant-box](https://github.com/AntonioMeireles/boot2docker-vagrant-box)**. The **matching up to date boxes are [here](https://atlas.hashicorp.com/AntonioMeireles/)**.
